Makefile

```diff
@@ -1,4 +1,3 @@
-CONTAINER_TOOL ?= podman
```

Comment:
(Optional) I might prefer this stay in place for flexibility if someone has docker, but it looks like you're removing it for consistency with the script, which is good. That said, the script could alternatively have a `CONTAINER_TOOL=${CONTAINER_TOOL:-podman}`.

Reply:
My understanding was that multiarch builds were defined differently in each tool. Docker does something with `buildx`, while podman uses the `manifest` commands seen in the scripts now. It would be more trouble, and I think it would be unnecessary, to support both tools.